src/view/com/notifications/Feed.tsx

```diff
@@ -170,6 +181,8 @@ export function Feed({
         contentContainerStyle={s.contentContainer}
         // @ts-ignore our .web version only -prf
         desktopFixedHeight
+        initialNumToRender={initialNumToRender}
+        windowSize={11}
```

Comment:
We were missing some `FlatList` perf props here. Removed the above comment after adding this, since I wasn't even getting the warning previously anyway.